chain/network/src/peer_manager.rs

```diff
@@ -2416,12 +2420,18 @@ impl Handler<RoutedMessageFrom> for PeerManagerActor {
             // Handle Ping and Pong message if they are for us without sending to client.
             // i.e. Return false in case of Ping and Pong
             match &msg.body {
-                RoutedMessageBody::Ping(ping) => self.handle_ping(ctx, ping.clone(), msg.hash()),
-                RoutedMessageBody::Pong(pong) => self.handle_pong(ctx, pong.clone()),
+                #[cfg(feature = "adversarial")]
+                RoutedMessageBody::Ping(ping) => {
+                    self.handle_ping(ctx, ping.clone(), msg.hash());
+                    false
+                }
+                #[cfg(feature = "adversarial")]
+                RoutedMessageBody::Pong(pong) => {
+                    self.handle_pong(ctx, pong.clone());
+                    false
+                }
```

Comment:
Does this actually make sense? Since we did not change `RoutedMessageBody` people can still send `Ping` and `Pong` message and shouldn't we return `false` for those?

Reply:
@bowenwang1996 That is a good question.  There is no code path to send `Ping` in production. Though, RoutedMessageBody::Ping / RoutedMessageBody::Pong are already in production, we have code which processes them, so we can't simply remove those messages. Only a peer that runs a modified code could use `ping`/`pong` messages. I don't think we care about those.

Reply:
I changed PR, unfortunately we have to keep following test messages, because they are part of protocol now, and we can't  remove test fields anymore, because they aren't at the end of enum.
```
                        | NetworkRequests::PingTo(_, _)
                        | NetworkRequests::FetchPingPongInfo
                        | NetworkRequests::FetchRoutingTable 
```

Reply:
hmm the code here still remains unchanged it seems?

Reply:
I wasn't sure whenever making `NetworkRequests` enums is fine. I checked the code, you are right, we don't use it as part of protocol. I put this put under `adversarial`.